source/type/message.cpp

```diff
@@ -329,6 +329,17 @@ void DisruptionHolder::clean_weak_impacts(){
     clean_up_weak_ptr(weak_impacts);
 }
 
+void DisruptionHolder::forget_vj(const VehicleJourney* vj) {
+    for (const auto& impact: vj->meta_vj->get_impacts()) {
+        for (auto& stu: impact->aux_info.stop_times) {
+            if (stu.stop_time.vehicle_journey == vj) {
+                std::cout << "problem with impact " << impact->uri << " and vj " << vj->uri << std::endl;
```

Comment:
:no_good:

Reply:
No I just need to remove this debug print.